snippets/numpy/core/fromnumeric.py

```diff
@@ -0,0 +1,78 @@
+
+from .overrides import set_module
+from .multiarray import asarray
+from ulab import numpy as np
+from ... import numpy
+
+def prod(arr):
+    result = 1
+    for x in arr:
+        result = result * x
+    return result
+
+def size(a, axis=None):
```

Comment:
I have meant to add this function for a long time, but somehow it slipped off my radar. Do you think it would be a good time now?

Reply:
Absolutely.